evergreen/client.go

```diff
@@ -1,4 +1,4 @@
-package cost
+package evergreen
```

Comment:
Do we need this still?

Reply:
It's required for the historical test stats job.